drake/systems/frames/test/CoordinateFrameCoordinatesTest.m

```diff
@@ -0,0 +1,64 @@
+classdef CoordinateFrameCoordinatesTest < matlab.unittest.TestCase
+  %NOTEST
+  
+  % Tests that functions that touch the coordinates property of
+  % CoordinateFrame work.
+  
+  properties
+    SpecifiedFrame
+    GeneratedFrame
+  end
+  
+  methods(TestMethodSetup)
+    function createFrames (tc)
+      % SpecifiedFrame has coordinate names set by the user directly
+      tc.SpecifiedFrame = CoordinateFrame('test1', 4, 'y', {'a', 'b', 'c', 'd'});
+      
+      % GeneratedFrame has coordinate names that are
+      % automatically generated from the prefix provided.
+      tc.GeneratedFrame = CoordinateFrame('test2', 4, 'z');
+    end
+  end
+  
+  methods(Test)
+    
+    function testGetCoordinateName (tc)
```

Comment:
yes - every static method for classes in a test directory will be run as tests (unless they are labeled with NOTEST)